spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplateTests.java

```diff
@@ -191,23 +191,16 @@ public void testSend_onFailure() {
 				this.pubSubTemplate.publish("testTopic", this.pubsubMessage);
 		this.settableApiFuture.setException(new Exception("future failed."));
 
-		try {
-			future.get();
-			fail("Test should fail.");
-		}
-		catch (InterruptedException ie) {
-			fail("get() should fail with an ExecutionException.");
-		}
-		catch (ExecutionException ee) {
-			assertEquals("future failed.", ee.getCause().getMessage());
-		}
+		assertThatThrownBy(() -> future.get())
```

Comment:
Consider to use this also instead of `ExpectedException`. Not the part of this PR though, but this is going to be a good move to JUnit 5 in the future.

Reply:
Acknowledged. Will move those instances to assertj as well moving forward.